chapters/2-document-creation-information.md

```diff
@@ -38,6 +38,22 @@ This specification uses the prefix `rdf:` to refer to the [RDF/XML][rdf] namespa
 http://www.w3.org/1999/02/22-rdf-syntax-ns#
 ```
 
+**2.1.7** YAML: `spdxVersion:`
+
+Example:
+
+```text
+spdxVersion: "SPDX-2.2"
```

Comment:
Does typical YAML usage require quotes around strings? I've seen it with or without, but I'm not that familiar with YAML so not sure what the norm is.

Reply:
It depends on what kind of scalar it is. If it is ASCII, it doesn't need to be in quotes. If it is double quoted, many parsers will decode it using UTF-8. If this is just a string, then it's best to remove the double quotes.